azext_iot/_help.py

```diff
@@ -288,21 +302,37 @@
     long-summary: For more information, see
                   https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities
     examples:
+    - name: Import all device identities from a blob by providing command parameters for
+            input blob container and storage account as well as output blob container and storage account.
+      text: >
+        az iot hub device-identity import -n {iothub_name} --ibc {input_blob_container_name} --isa {input_storage_account_name}
+        --obc {output_blob_container_name} --osa {output_storage_account_name}
```

Comment:
Should we state what would be the name + purpose of the output blob that will be produced from an import device identity job?